prefixdb_test.go

```diff
@@ -1,6 +1,12 @@
 package db
```

Comment:
As written, this test can't run unless all the build tags are enabled. I think we should split out the tests that require specific build tags (e.g., CLevelDB, RocksDB, BadgerDB, BoltDB) into tagged files so they only run with those tags enabled. Otherwise even if someone doesn't want to use those backends, they are forced to install all of the support libraries just to run the unit tests.

Reply:
OK, I thought about doing that, can do that now...